doc/holochain_101/src/zome/intro_to_hdk.md

```diff
@@ -0,0 +1,55 @@
+# Intro to Language HDKs
+
+Within any Zome, there are certain things that must be implemented within the code, in order to work with Holochain. For each language that Zome code can be written in, like Rust, there should be a software library for that language to make it as easy as possible to define those standard functions and behaviours. We call this a "Holochain Development Kit", or HDK.
+
+So in order to get familiar with coding for Holochain, it will involve familiarity with an HDK library.
+
+An HDK should perform many important functions for developers.
```

Comment:
I would prefer removing the "should perform" formulation to the simpler "performs"

Reply:
Makes sense.